docs/src/examples/components/Dropdown/Types/DropdownExampleSingleSelection.shorthand.steps.ts

```diff
@@ -0,0 +1,13 @@
+import { Dropdown, Button, List } from '../../../../../../src'
```

Comment:
```suggestion
import { Dropdown, Button, List } from '@stardust-ui/react'
```

Can we use just this?

If it requires `tsconfig`, this should work: 
```ts
require("ts-node").register({
  project: "build/tsconfig.common.json",
});
```

Reply:
let me check..

Reply:
unfortunately, this doesn't help - but, actually, I would also expect that it should. Will take another look tomorrow, thank you!

Reply:
taking this as a general problem (import expressions like this one could be met quite often in the code), will clean up this stuff in the dedicated PR. Currently wasn't able to come up with resolution, but will spend additional time today